### DISCUSSION TITLE
Multiple packageClassList

### DISCUSSION THREAD

**Discussion by alexookah** [capawesome-team/capacitor-plugins#196] [Q&A]
![Screenshot 2024-06-12 at 14 05 22](https://github.com/capawesome-team/capacitor-plugins/assets/9624843/c1cef29c-641f-4bb8-8091-90819f6d1c13)

I have in my capacitor.config.json multiple same classList names
```
		"FirebaseAuthenticationPlugin",
		"FirebaseAuthenticationPlugin",
		"FirebaseMessagingPlugin",
		"FirebaseMessagingPlugin",
		"LiveUpdatePlugin",
		"LiveUpdatePlugin"
```

I am not sure how this happened. It should be unique, right?

**Comment by robingenz** [comment]
You need to update to the latest Capacitor version. I close this as duplicate of https://github.com/ionic-team/capacitor/pull/7470.
